src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = new Lottery[] {
+                new Lottery(),
+                new Lottery(),
+                new Lottery()
+        };
+        for (var b : balls) {
+            System.out.println(b.getRandomBall().toString());
+        }
```

Comment:
You don't need to create 3 Lottery objects, because ONE lottery can produce all of 3 balls :)